docs/migrating.md

```diff
@@ -30,6 +42,19 @@ export default defineConfig({
 })
 ```
 
+#### Output
+The output provides a description of the necessary changes or removals required when migrating from Fabric to Warp. The process requires to be done manually, so make sure that these modifications are applied to all relevant class names and component names.
+
+```js
+[REPLACED] text-12
+[REPLACED] text-blue-600
+[REPLACED] text-gray-400
```

Comment:
Just because docs for those classes are missing for now, maybe you could add these examples instead 😄 
```
[REPLACED] flex-shrink -> use shrink
[REPLACED] flex-grow -> use grow
[REPLACED] last-child:mb-0 -> use last:mb-0
```

Reply:
haha very true